test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,50 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const query = {
```

Comment:
as you are reusing this en several endpoints, not always is sent as a query, so rename as data, or something similar